documentation/latest/intro/overview.md

```diff
@@ -16,40 +16,8 @@ It is also possible to schedule a container workload to run in the device using
 a Workload CRD. A container workload can be scheduled for an unlimited
 number of devices.
 
-{% mermaid %}
-flowchart TB
-  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
-  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
-  classDef cluster fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+![Overview](../../../assets/images/overview.png?raw=true)
```

Comment:
But with mermaid, I do not know how to address this `Diagrams aren't aligned and has wrong proportions to the elements` or maybe I misunderstood the remark
@masayag can you provide feedback on that?

Reply:
I'm happy to build that on Mermaid

Reply:
I agree with Eloy about reducing the maintenance and contribution complexity.
I'm not a mermaid expert, but I like the simplicity and the lack of need to maintain rendered resources stored in png format when it can be avoided.

For this case, let's drop this form this PR and address it in a separate PR, since the rest seems ready to be merged.

Next, we can think of changing the type of the diagram from flowchart to [crc](https://mermaid-js.github.io/mermaid/#/c4c) and also to rework the content as well the positioning within this diagram (let the management be either on the top/north side and device on the south or even mgmt on the left, device on the right and we can omit the ingress). but let's have this outside of the context of this PR, not to block it.

Reply:
I removed the commit and I created a ticket to address to rework of all diagrams :)